### PR TITLE
disable autocorking

### PR DESCRIPTION
**Pull request by salvatoredipietro**
Update optimization_recommendation.md with recommendation to disable autocorking

### CODE REVIEW

File: perfrunbook/optimization_recommendation.md [hunk 1]
```diff
@@ -55,6 +55,7 @@ A TLB (translation lookaside buffer) is a cache that holds recent virtual addres
 3. Disable Receive Packet Steering (RPS) to avoid contention and extra IPIs. 
     1.  `cat /sys/class/net/ethN/queues/rx-N/rps_cpus` and verify they are set to `0`. In general RPS is not needed on Graviton2. 
     2. You can try using RPS if your situation is unique.  Read the [documentation on RPS](https://www.kernel.org/doc/Documentation/networking/scaling.txt) to understand further how it might help. Also refer to [Optimizing network intensive workloads on Amazon EC2 A1 Instances](https://aws.amazon.com/blogs/compute/optimizing-network-intensive-workloads-on-amazon-ec2-a1-instances/) for concrete examples.
+4. Disable `tcp_autocorking` with `echo 0 > /proc/sys/net/ipv4/tcp_autocorking` and use `TCP_NODELAY` socket flag to avoid addition delays in the server's response that could impact latency
```
Comment (wash-amzn) [inline perfrunbook/optimization_recommendation.md:58]: (sorry) The other items end in a period

Comment (salvatoredipietro) [reply perfrunbook/optimization_recommendation.md]: added it

File: perfrunbook/optimization_recommendation.md [hunk 2]
```diff
@@ -55,7 +55,7 @@ A TLB (translation lookaside buffer) is a cache that holds recent virtual addres
 3. Disable Receive Packet Steering (RPS) to avoid contention and extra IPIs. 
     1.  `cat /sys/class/net/ethN/queues/rx-N/rps_cpus` and verify they are set to `0`. In general RPS is not needed on Graviton2. 
     2. You can try using RPS if your situation is unique.  Read the [documentation on RPS](https://www.kernel.org/doc/Documentation/networking/scaling.txt) to understand further how it might help. Also refer to [Optimizing network intensive workloads on Amazon EC2 A1 Instances](https://aws.amazon.com/blogs/compute/optimizing-network-intensive-workloads-on-amazon-ec2-a1-instances/) for concrete examples.
-4. Disable `tcp_autocorking` with `echo 0 > /proc/sys/net/ipv4/tcp_autocorking` and use `TCP_NODELAY` socket flag to avoid addition delays in the server's response that could impact latency
+4. Disable `tcp_autocorking` with `echo 0 > /proc/sys/net/ipv4/tcp_autocorking` and use `TCP_NODELAY` socket flag to avoid addition delays in the server's response that could impact latency.
```
Comment (wash-amzn) [inline perfrunbook/optimization_recommendation.md:58]: s/addition/additional/

**Review by wash-amzn — APPROVED** [review]

**Comment by wash-amzn** [comment]
Too late since it's merged, but I should have asked you to squash your commits, 3 commits to add a single sentence isn't good for commit history ;)

**Pull request merged**